src/aat/resources/features/F-027/F-027.feature

```diff
@@ -76,6 +76,20 @@ Scenario: must return appropriate negative response for a user not having a prof
     And the response [has the 403 return code],
     And the response has all other details as expected.
 
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+@S-039
+Scenario: must return appropriate negative response for invalid caseType
+
+    Given a user with [an active profile in CCD],
+
+     When a request is prepared with appropriate values,
+      And the request [does not provide valid caseType],
```

Comment:
"not providing a valid input" and "providing an invalid input" are not same things. pls see comment above for the same line.